packages/ui/src/components/select/select.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { cx } from "classix";
+import React, { ComponentProps } from "react";
+import {
+  Button as BaseButton,
+  FieldError as BaseFieldError,
+  Form as BaseForm,
+  Label as BaseLabel,
+  ListBox as BaseListBox,
+  ListBoxItem as BaseListBoxItem,
+  Popover as BasePopover,
+  Select as BaseSelect,
+  SelectValue as BaseSelectValue,
+} from "react-aria-components";
+
+export const Form = (properties: ComponentProps<typeof BaseForm> & { className?: string | undefined }) => {
+  const { className = "", ...rest } = properties;
+  const mergedClassName = cx("relative w-full", className);
+  return <BaseForm className={mergedClassName} {...rest} />;
+};
+
+export const Select = (
```

Comment:
Since comparing our solution with shadcn, I think we should include React forwardRef and React displayName to go for an A+. e.g.:

```tsx
export const Select = React.forwardRef<
  React.ElementRef<typeof SelectPrimitive>,
  React.ComponentPropsWithoutRef<typeof SelectPrimitive>
>((properties, ref) => {
  const { className = "", ...rest } = properties;
  const mergedClassName = cx("group w-full focus:outline-none", className);
  return <SelectPrimitive className={mergedClassName} {...rest} />;
});

Select.displayName = SelectPrimitive.displayName;
```

They refer to `BaseSelect` as `SelectPrimitive` which is better naming IMO.